vm/src/bytesinner.rs

```diff
@@ -263,6 +263,8 @@ impl PyBytesInner {
         op: PyComparisonOp,
         vm: &VirtualMachine,
     ) -> PyComparisonValue {
+        // TODO: bytes can compare with any object implemented buffer protocol
+        // but not memoryview, and not equal if compare with unicode str(PyStr)
         PyComparisonValue::from_option(
             try_bytes_like(vm, other, |other| {
                 op.eval_ord(self.elements.as_slice().cmp(other))
```

Comment:
Now we have different behavour between (memoryview op bytes) and (bytes op memoryview), Is any solution better than check the type here?

Reply:
Does memory view not implement the buffer protocol? I think that would maybe fix the issue(?)

Reply:
But memoryview is implemented buffer protocol, we will have much more issue if not so.

Reply:
Huh, that's strange, I think that's probably fine for now.